default.yml

```diff
@@ -78,8 +78,11 @@ Style/WordArray:
 Layout/SpaceAfterComma:
   Enabled: true
 
-Layout/SpaceInsideBrackets:
-  Enabled: false
+Layout/SpaceInsideReferenceBrackets:
+  Enabled: true
+
+Layout/SpaceInsideArrayLiteralBrackets:
+  Enabled: true
```

Comment:
Is this an actual change to our rules?

Reply:
Hmm yes... 
The old rule enforced `[1]` instead of `[ 1 ]`. The new rules can be configured (in the case of array literal `[1]` or bracket reference `flash[:notice]`).
A large majority of our usage in glue is without the spaces, but there are some places where we do have them.
I'll revert this change.